sass/style.sass

```diff
@@ -0,0 +1,131 @@
+@import "reset"
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;400;600&display=swap')
+@import '../fonts/scss/fontawesome'
+@import '../fonts/scss/solid'
+@import '../fonts/scss/brands'
+@import '../fonts/scss/regular'
+
+@import "variables"
+@import "colors"
+
+
+body
+  font-family: $font-main
+
+  .portfolio-body
+    height: 100%
+    background-color: $bck-clr
+    padding: 20px
+    display: block
+
+.new-footer
+  display: block
```

Comment:
is it necessary? Could you inspect your styles and remove redundant `display: block`?